verticapy/toolbox.py

```diff
@@ -1139,22 +1159,25 @@ def replace_external_queries_in_query(query: str):
         "alter ",
         "update ",
     )
-    external_queries = re.findall("\$\$\$(.*?)\$\$\$", query)
-    for idx, external_query in enumerate(external_queries):
-        if external_query.strip().lower().startswith(sql_keyword):
-            external_query_tmp = external_query
-            subquery_flag = False
-        else:
-            external_query_tmp = f"SELECT * FROM {external_query}"
-            subquery_flag = True
-        query_dblink_template = get_dblink_fun(external_query_tmp)
-        if subquery_flag:
-            if " " in external_query.strip():
-                alias = f"VERTICAPY_EXTERNAL_TABLE_{idx}"
+    for s in verticapy.options["external_connection"]:
```

Comment:
not sure about this. 

don't you want to get the symbol inside the connection e.g. vertica.options["external_connection"["symbol"]

Reply:
verticapy.options["external_connection"] is a dict of symbols